.version

```diff
@@ -1 +1 @@
-1.16.0
+1.17.0
```

Comment:
This doesn't seem right. The master branch is already at 1.17.0: https://github.com/Cray-HPE/hms-firmware-action/blob/master/.version

Need to bump the version I would think, since we are getting our base images from somewhere else. I guess you could make the argument that we don't need to, since its basically the same base alpine version from a different source and won't take effect until we make a bugfix/feature change to FAS when we create a Git tag/release.

Reply:
ok; updated.